### PR TITLE
Update chart and documentation for brig proxy

### PR DESCRIPTION
**Pull request by radu-matei**
- update the default chart service type to `ClusterIP`
- update the NOTES.txt message if the service type is `ClusterIP` to use `brig proxy`
- update the install and developer documentation

### CODE REVIEW

File: docs/developers.md [hunk 1]
```diff
@@ -38,16 +38,10 @@ image:
 ```console
 $ eval $(minikube docker-env)
 $ yarn docker-build
-$ helm install -n brigade-ui chart/kashti --set brigade.apiServer=http://localhost:7745
+$ helm install -n kashti ./charts/kashti
```
Comment (seanknox) [inline docs/developers.md]: Nice removal of the default value.

File: docs/install.md [hunk 2]
```diff
@@ -4,76 +4,28 @@ This document covers the installation of Kashti into a Kubernetes cluster. Kasht
 developers will find developer-specific documentation in the [Developer Guide](developers.md).
 
 Kashti is a client-side JavaScript application. It makes requests to the Brigade API
-server (which is installed as part of Brigade). **The browser that Kashti is run from
-_must_ have access to the Brigade API server.**
+server (which is installed as part of Brigade). We will use the `brig` command line to create HTTP tunnels to the Brigade API server and to the Kashti dashboard.
+
+> If you don't have the `brig` command line already, [check out the Brigade documentation on how to build it](https://github.com/Azure/brigade/blob/master/docs/topics/developers.md#building-source)
```
Comment (seanknox) [inline docs/install.md]: Handy! Thanks for adding a brig link.

**Comment by radu-matei** [comment]
Thanks for reviewing - this can be merged, right?

**Comment by seanknox** [comment]
Yup! If either Matt/Ronan/I approve you are good to go. :shipit:

**Comment by radu-matei** [comment]
Yeah, but I can't merge

**Pull request merged**